components/HTMLContent.js

```diff
@@ -64,7 +64,7 @@ const HTMLContent = styled(({ content, collapsable = false, maxCollapsedHeight =
   const DisplayBox = !isCollapsed || isOpen ? InlineDisplayBox : CollapsedDisplayBox;
 
   useEffect(() => {
-    if (collapsable && contentRef?.current?.clientHeight > maxCollapsedHeight + 1) {
+    if (collapsable && contentRef?.current?.clientHeight > maxCollapsedHeight * 3) {
```

Comment:
I don't get the logic of this line: "If the content is three times taller than the max, collapse it". If I set a **max** height, I expect the content to not go beyond this value. Like if I set a `maxHeight` of 20px, I expect the content to be collapsed when it's 21px, not 60px. Why this `* 3`?

Reply:
So if the user enters text that only spans one line we just show it without any collapsing. But if the user inserts text that spans two lines then the problem is we will still _collapse_ it. What I mean by "collapse" is that we will show the second line with blurred text and how the expand/collapse link. 

This is a problem. In fact when the user enters text that spans two lines, we should check if the third line exists and only then we should collapse. Note that we are blurring text. So if we just "collapse" in the second line, we show the first line and "blur" the second line, thus showing both lines and making the collapse link useless really. The user can read both lines. One is fully visible, the second line blurred. This problem can be seen in the screenshot; 

<img width="741" alt="Screen Shot 2021-11-03 at 1 59 16 PM" src="https://user-images.githubusercontent.com/5498878/139973874-9543f799-c92c-4fb6-b90a-608bb0a9ad88.png">

My thinking is to thus take into account the third line; the second line is visible (just blurred); hence the above solution of `*3`. Does that makes sense, let me know otherwise I might not have explained this correctly? 🤔

Reply:
But do you see how it creates a strange API? Let's say I have a design to implement with a text that should be collapsed from `30px` height and above. My naive implementation would be to write:

```jsx
<HtmlContent maxCollapsedHeight={30} />
```

But if I do that, the component will - by default - grow to 40px, 50px and up to 90px. So I'll have to divide the value I'm passing by 3:
```jsx
<HtmlContent maxCollapsedHeight={10} />
```

If I set a **max** height, I expect the content to not go beyond this value. It looks to me that we can solve opencollective/opencollective#4903 by simply changing the value we're padding to `maxCollapsedHeight` for expense items. If not, we should try to find a more predictable API for this component.

Reply:
That makes sense; thanks for the explanation. I've added an additional prop `collapsePadding` to handle this problem. With this the we can specify exactly how much to pad. Let me know how it looks now. Here's a screenshot of the effect; 

![image](https://user-images.githubusercontent.com/12435965/140974381-387ace4a-612c-4963-b180-1442e15f5afe.png)

Note that in the first expense description there's only two lines so it's not blurred while in the second there's more than three lines.